docs/migrating-from-0.5.md

```diff
@@ -2,7 +2,127 @@
 
 Genkit 0.9 introduces a number of breaking changes alongside feature enhancements that improve overall functionality. If you have been developing applications with Genkit 0.5, you will need to update your application code when you upgrade to the latest version. This guide outlines the most significant changes and offers steps to migrate your existing applications smoothly.
 
-## 1. CLI Changes
+## Quickstart guide
+
+The following steps will get you up and running on Genkit 0.9 quickly. Read more information about these changes in the detailed [Changelog](#changelog) below.
+
+### 1. Install the new CLI
+
+* Uninstall the old CLI
+
+```posix-terminal
+npm uninstall -g genkit
+```
+
+* Install the new CLI
+
+```posix-terminal
+# Install globally
+npm i -g genkit-cli
+
+# Or install as a dev dependency
+npm i -D genkit-cli
+```
+
+### 2. Update your dependencies
+
+* Remove individual Genkit core packages
+
+```posix-terminal
+npm uninstall @genkit-ai/ai @genkit-ai/core @genkit-ai/dotprompt @genkit-ai/flow
+```
+
+* Install the new consolidated `genkit` package
+
+```posix-terminal
+npm i --save genkit
+```
+
+* Upgrade all plugin versions (example below)
+
+```
+npm upgrade @genkit-ai/firebase
```

Comment:
FWIW this didn't work for me, maybe because of the `rc` branch.  Could we make this copy-pastable for all the supported plugins? It looks to be a no-op if the package isn't in `package.json`

Reply:
It also didn't work for me. I think this is because it isn't released yet. I'm hesitant to change it until we know whether that is true.

Reply:
Let's tackle this in a followup.